.github/workflows/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Main
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        node_version:
+          - 12
+          - 14
+          - 16
+    name: Node ${{ matrix.node_version }} on ${{ matrix.os }}
+    steps:
+      - name: Use LF EOL
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: |
+          git config --global core.autocrlf false
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node_version }}
+      - name: Test
+        run: |
```

Comment:
Do we need:

```yaml
if: ${{ matrix.os != 'windows-latest' }}
```

here?

Reply:
That would satisfy the direction of doing "switch to only doing yarn test on Mac + linux and yarn test-ie on Windows". I'll put that in.